kytos/core/api_server.py

```diff
@@ -23,7 +27,7 @@ def __init__(self, app_name):
         self.flask_dir = os.path.join(dirname, '../web-ui/source')
 
         self.app = Flask(app_name, root_path=self.flask_dir)
-        self.server = SocketIO(self.app)
+        self.server = SocketIO(self.app, async_mode='threading')
```

Comment:
How the API server is running in thread, we need to use `async_mode` in SocketIO server, to start and shutdown the server.